src/hooks/useActiveWorkspaceFromNavigationState.ts

```diff
@@ -18,10 +18,10 @@ function useActiveWorkspaceFromNavigationState() {
             Log.warn('useActiveWorkspaceFromNavigationState should be called only from BottomTab screens');
         }
 
-        const params = state.routes.at(-1)?.params ?? {};
+        const lastHomeParams = state.routes.findLast((route) => route.name === SCREENS.HOME)?.params ?? {};
```

Comment:
Hi! That's a good question. Could you please be a little more specific about the case when it could break? I am not sure if I understand it correctly. Thanks! 🙇